frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -748,6 +748,7 @@ end
 
 function ReaderHighlight:onShowHighlightMenu()
     if not self.selected_text then
+        logger.dbg("readerhighlight: onShowHighlightMenu called with nil selected_text -- ignoring")
         return
```

Comment:
Cf. https://github.com/koreader/koreader/commit/a7cfc659288c7dbd9ffbe9b214f9980e6dc429ee

Reply:
I was just thinking whether `dbg:...` would be preferred. Will switch.